packages/file-picker/ios/Plugin/FilePickerPlugin.swift

```diff
@@ -13,10 +13,12 @@ public class FilePickerPlugin: CAPPlugin {
     public let errorFileNotExist = "File does not exist."
     public let errorConvertFailed = "File could not be converted."
     public let errorPickFileCanceled = "pickFiles canceled."
+    public let errorPickDirectoryCanceled = "pickDirectory canceled."
     public let errorUnknown = "Unknown error occurred."
     public let errorTemporaryCopyFailed = "An unknown error occurred while creating a temporary copy of the file."
     public let errorUnsupportedFileTypeIdentifier = "Unsupported file type identifier."
     public let pickerDismissedEvent = "pickerDismissed"
+    public var invokedMethod: String?
```

Comment:
Let's move this property into `packages/file-picker/ios/Plugin/FilePicker.swift` since it's only needed there.